home/home_2.1/main.swift

```diff
@@ -0,0 +1,119 @@
+import Foundation
+
+let FILENAME = "user_stats.txt"
+let DELETE_COMMAND = "delete"
+let BREAD_COMMAND = "bread"
+let EXIT_COMMAND = "exit"
+
+struct StatsManager {
```

Comment:
I hope that it may be helpful for you to read a bit about access control specifiers in swift like public, private and internal - you can practice them on this class. You can find some basic info 
[here](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/accesscontrol/) and [here](https://www.codecademy.com/learn/learn-swift-properties-and-access-control/modules/swift-properties-and-access-control/cheatsheet)